napari/_qt/qt_main_window.py

```diff
@@ -949,12 +941,9 @@ def screenshot(self, path=None):
 
     def close(self):
         """Close the viewer window and cleanup sub-widgets."""
-        try:
-            if hasattr(self, '_qt_window'):
-                self._qt_window.close()
-                del self._qt_window
-        except AttributeError:
-            pass
+        if hasattr(self, '_qt_window'):
+            self._qt_window.close()
+            del self._qt_window
```

Comment:
I think I was getting an error here with the del, if it was already deleted and that is why OI left a try/except there.

Reply:
yeah, I went back to it too. thanks

Reply:
(our viewer.window object is in a strange state post-close, and needs help in general)